frontend/revive-gallery-app/src/components/registrationform.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+import "./loginform.css";
+
+const RegistrationForm = () => {
+    const [registration, setRegistration] = useState({
+        first_name: '',
+        last_name: '',
+        username: '',
+        email: '',
+        password: ''
+    });
+
+    const [popupStyle, showPopup] = useState("hide");
+
+    const popup = () => {
+        // Send the data to the backend API
+        fetch("http://localhost:5000/register", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify(registration),
+        })
+            .then((response) => {
+                if (response.ok) {
+                    showPopup("signup-popup");
```

Comment:
Can you please redirect to the home page after successful login ?

Reply:
Yes, I will do it once I created the UI for home page

Reply:
Okay ! Thank you